tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27-django111,py27-django111-check,docs,flake8
+envlist = py27-django111,py27-django111-check,py3-django2,py3-django2-check,docs,flake8
```

Nit: Not sure if it works though.

```suggestion
envlist = py{27-django111,3-django2}{,-check},docs,flake8
```

Works in bash.
```bash
$ echo py{27-django111,3-django2}{,-check}
py27-django111 py27-django111-check py3-django2 py3-django2-check
```